mtv_pipelines/wrappers/jenkins_analyzer.py

```
@@ -16,59 +16,42 @@
-                },
-                verify=False,
-            )
-            resp.raise_for_status()
-            data = json.loads(resp.content)
-            analysis = self._process_data(data, job_result)
-
-            return analysis
-        return JenkinsJobAnalysisDTO(
-            job_result=job_result, summary="", child_jobs=[], html_report_url=""
+        logger.info(f"Analyzing job result for {job_result.url}")
+        resp = requests.post(
```

Ruff (0.16.0)

[error] Probable use of `requests` call without timeout

(S113)